### DISCUSSION TITLE
直接复制的默认密码为什么提示密码错误？

### DISCUSSION THREAD

**Discussion by i55top** [AlistGo/alist#2325] [Q&A]
INFO[2022-11-14 06:49:21] reading config file: data/config.json        
INFO[2022-11-14 06:49:21] load config from env with prefix: ALIST_     
INFO[2022-11-14 06:49:21] init logrus...                               
INFO[2022-11-14 06:49:21] admin user's info: 
username: admin
password: 3pADXTIO

**Comment by zxx457** [comment]
你用alist server , 默认会读取 data/config.json这个配置, 密码就是这个配置里面的

**Comment by lusirnoman** [comment]
我也遇到这个问题了
**实际上应该进入到/opt/alist目录执行 sudo ./alist admin**
而不是在安装目录执行，安装目录的data文件夹和/opt/alist下的文件夹不一样

**Reply by SoolyChristy** [reply]
我只有/alist这个目录啊

**Reply by Abinthur** [reply]
same issue here.

**Comment by smgcd** [comment]
我的是，data文件夹里的config文件里的user和password没了，得自己加上去

**Reply by xhofe** [reply]
……不要误导，data文件夹里的config文件里的user和password是使用除sqlite3之外的数据库的数据库的密码。

**Comment by HKL-Kevin** [comment]
> INFO[2022-11-14 06:49:21] reading config file: data/config.json INFO[2022-11-14 06:49:21] load config from env with prefix: ALIST_ INFO[2022-11-14 06:49:21] init logrus... INFO[2022-11-14 06:49:21] admin user's info: username: admin password: 3pADXTIO

看你的alist守护进程是怎么启动的，加上相应参数就好了。
如我的是：
<img width="769" alt="image" src="https://github.com/alist-org/alist/assets/44154392/af4177e8-9a65-4f2a-871f-520ef13cb2a1">
那么查看密码的命令就是：
alist -data /etc/alist --no-prefix admin

**Reply by aiaimimi0920** [reply]
你的答复切实且有效。

这里给个总的解决方法，

1. sudo -i
2. ps -ef | grep alist
你会得到类似这样的东西：
alist3    9668     1  0 15:54 ?        00:00:00 /volume1/@appstore/alist3/bin/alist server --data /volume1/@appdata/alist3

3.  cd  /volume1/@appstore/alist3/bin/
4.  ./alist admin set 你的密码 --data /volume1/@appdata/alist3

**Reply by chinndesu** [reply]
正解，我还打算编辑data.db手动修改password...
